dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganisationResource.java

```diff
@@ -89,7 +89,7 @@ public Organisation approveOrganisation(@ApiParam(hidden = true) @Auth User user
         }
         organisation.setApproved(true);
 
-        Event createOrganisationEvent = new Event(null, organisation, null, null, user, Event.EventType.APPROVE_ORG);
+        Event createOrganisationEvent = new Event(null, organisation, null, null, null, user, Event.EventType.APPROVE_ORG);
```

Comment:
Better to use a builder pattern here (and for other code calling Event constructor), this is going to quickly get unwieldy (or is already for that matter).